File: truth_set/python/logic/wrong_location/dataFetcher.py
import asyncio

class UserProfile:
    data = None
    def __init__(self):
        asyncio.create_task(self.fetch_data())
    
    async def fetch_data(self):
        await asyncio.sleep(1)
        self.data = {"name": "Alice"}
        print(f"Data fetched: {self.data}")
    
    def print_data(self):
        print(f"User Data: {self.data}")

async def main():
    profile = UserProfile()
    profile.print_data()

asyncio.run(main())
\ No newline at end of file

File: truth_set/go/logic/wrong_location/dataFetcher.go
package wrong_location

import (
    "fmt"
    "time"
)

type UserProfile struct {
    data map[string]string
}

func NewUserProfile() *UserProfile {
    up := &UserProfile{}
    go up.fetchData() 
    return up
}

func (up *UserProfile) fetchData() {
    time.Sleep(1 * time.Second)
    up.data = map[string]string{"name": "Alice"}
    fmt.Println("Data fetched:", up.data)
}

func (up *UserProfile) PrintData() {
    fmt.Println("User Data:", up.data)
}

func dataFetcherMain() {
    profile := NewUserProfile()
    profile.PrintData()
}
\ No newline at end of file